src/Tickets/Emails/Dispatcher.php

```diff
@@ -552,9 +553,12 @@ public function send(): bool {
 			return false;
 		}
 
+		// Handle any encoded characters in the subject.
+		$subject = wp_specialchars_decode( $this->get_subject() );
```

Comment:
I wonder if we should move the escaping to the method itself.

Reply:
I had thought about that, but because of the filters in that method, I felt we wanted to decode the string really late. That's when I figured we'd just decode the string right before it's sent. That guarantees that it will be decoded.